web/src/styles.css

```diff
@@ -1390,6 +1402,128 @@ sub {
     width: 220px;
 }
 
+.zone-time-slider {
+    position: absolute;
+    bottom: 0;
+    height: 62px;
+    background: rgb(70, 70, 70);
+    width: 100%;
+    text-align: center;
+    box-shadow: 0 0 5px 1px rgba(0,0,0,0.3);
+    overflow-y: hidden;
+}
+
+.zone-time-slider .time-slider-axis-container{
+    width: 85%;    
+    height: 20px;
+    overflow: visible;
+}
+
+.time-slider-axis .tick {
+    font-size: 12px;
+}
+
+.zone-time-slider .domain {
+    display: none;
+}
+
+
+.time-slider-input {
+    -webkit-appearance: none;
+    width: 95%;
+    margin-top: 5px;
+    height: 26px;
+    background: transparent; 
+    padding: 0;
+    overflow: visible;
+  }
+  
+  .time-slider-input:focus {
+    outline: none;  
+  }
+  
+/* WebKit/Blink */
+
+.time-slider-input::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    border: 1px solid gray;
+    height: 26px;
+    width: 26px;
+    border-radius: 17px;    
+    background: #ffffff;
+    cursor: pointer;
+    margin-top: -12px;
+  }
+
+  .time-slider-input::-webkit-slider-runnable-track {
+    width: 100%;
+    margin-right: 10px;
+    margin-left: 10px;
+    height: 3px;
+    cursor: pointer;
+    background: lightgray;
+    border-radius: 1.3px;
+    border: 0.2px solid #010101;
+  }
+  
+  .time-slider-input:focus::-webkit-slider-runnable-track {
+    background: white;
+  }
+
+
+  /* IE */
```

Comment:
omg crazy IE style :(
Well done.
Can you maybe add a small comment on why IE and FF need separate styles?